Editor/Animation/AnimationDatabase.cs

```diff
@@ -80,10 +80,38 @@ public void SetCurrentNoInvalidate(Motion newMotion)
             }
         }
 
+#if MA_VRCSDK3_AVATARS_3_5_2_OR_NEWER
+        internal class PlayAudioHolder
+        {
+            private VRCAnimatorPlayAudio _currentPlayAudio;
+
+            internal VRCAnimatorPlayAudio CurrentPlayAudio
+            {
+                get
+                {
+                    return _currentPlayAudio;
+                }
+                set
+                {
+                    _currentPlayAudio = value;
+                }
+            }
+
+            internal PlayAudioHolder(VRCAnimatorPlayAudio audio)
+            {
+                CurrentPlayAudio = audio;
+            }
+        }
```

Comment:
We probably don't need a holder class for this and can just reference the VRCAnimatorPlayAudio assets directly. The reason we have the holder for clips is because we need to swap out clips for different clips in processing (in part due to the lack of an API to delete editor curve bindings), but with VRCAnimatorPlayAudio the state behaviors are already cloned at this point and we can just update the paths in place.